bridges/primitives/header-chain/src/justification/mod.rs

```diff
@@ -54,6 +54,17 @@ pub struct GrandpaJustification<Header: HeaderT> {
 	pub votes_ancestries: Vec<Header>,
 }
 
+// A proper Debug impl for no-std is not possible for the `GrandpaJustification` since the `Commit`
+// type only implements Debug that for `std` here:
+// https://github.com/paritytech/finality-grandpa/blob/8c45a664c05657f0c71057158d3ba555ba7d20de/src/lib.rs#L275
+// so we do a manual impl.
+#[cfg(not(feature = "std"))]
+impl<H: HeaderT> core::fmt::Debug for GrandpaJustification<H> {
+	fn fmt(&self, f: &mut core::fmt::Formatter) -> core::fmt::Result {
+		write!(f, "GrandpaJustification {{ round: {:?}, commit: <wasm:stripped>, votes_ancestries: {:?} }}", self.round, self.votes_ancestries)
+	}
+}
```

Comment:
Sadly not, since a few spots require `Debug` also for the `std` case. Honestly no clue how it could compile in the past.  

I think the fix is to use something like `RuntimeDebugNoBound` in the grandpa crate.